one-ui/src/main/java/com/marklogic/hub/oneui/managers/ModelManager.java

```diff
@@ -182,18 +182,6 @@ public JsonNode getLatestJobInfo(String modelName) {
     }
 
     JsonNode getJobInfoFromDB(DatabaseClient dbClient, String modelName) {
-        JsonNode node = null;
-        try {
-            node = JobInfo.on(dbClient).getLatestJobData(modelName);
-        }
-        catch (MarkLogicServerException e) {
-            /*
-             * As MarkLogic server throws a FailedRequestException for both data-service module not found
-             * and if the user is unauthorized to access the module; we need to fail silently and log the
-             * issue.
-             */
-            logger.error(e.getMessage());
-        }
-        return node;
+        return JobInfo.on(dbClient).getLatestJobData(modelName);
```

Comment:
I think the client should just call JobInfo then - there's no point in having this additional layer to go through. If we did keep it, I think we'd want the method names to be the same - i.e. the Manager method should be getLatestJobData.

Reply:
The separate method is used for mocking the database call in the ModelManagerTest class
```
doReturn(expectedNode).when(modelManager).getJobInfoFromDB(dataServiceClient, modelName);
```

I'll change the method name to `getLatestJobData`

Reply:
Can you mock ArtifactsService instead? That's supposed to be one of the benefits of data services - that there's an interface class that can be mocked for testing purposes.

Reply:
But - if it's easier to just mock a single thing - i.e. ModelManager - that's fine too.

Reply:
> Can you mock ArtifactsService instead? That's supposed to be one of the benefits of data services - that there's an interface class that can be mocked for testing purposes.

By ArtifactsService I am guessing you mean the JobInfo interface that was generated by the data service. 
JobInfo has a static `on` method which we call as 
```
JobInfo.on(finalDataServiceClient).getLatestJobData(s)
```
I believe mockito does not support mocking static stuff and would require us to use a testing framework like powermockito. 

Hence, in this case, it's simpler to just mock the `getLatestJobData`  method of ModelManager.